### ISSUE TITLE
strange css issue accessing ZAC

### ISSUE THREAD

**Issue by dovholuknf** [openziti/ziti-browzer-runtime#245]
i deployed the ZAC behind browzer and noticed some goofy styling on certain pages. some pages (most pages) are fine but some have goofed up css.

Notice how after 5 seconds "something" happens to the css and the spacing gets goofed up and styles seem off. Only seems to happen when run via browzer. The same page exposed via zrok operated fine

![goofy-zac](https://github.com/openziti/ziti-browzer-runtime/assets/46322585/4797f150-e37e-4a93-83df-42b3d724a4dd)

**Comment by rgallettonf** [comment]
After a little digging, I can see that BrowZer is appending it's own custom stylesheet to the page: `ziti-browzer-css.css`. One of the rules in that style sheet is using a **psuedo-element** style which is adding additional content after each `.row` element

```
.zitiBrowzerRuntimeSettings .btn-group-vertical>.btn-group:after, .btn-group-vertical>.btn-group:before, .btn-toolbar:after, .btn-toolbar:before, .clearfix:after, .clearfix:before, .container-fluid:after, .container-fluid:before, .container:after, .container:before, .dl-horizontal dd:after, .dl-horizontal dd:before, .form-horizontal .form-group:after, .form-horizontal .form-group:before, .modal-footer:after, .modal-footer:before, .modal-header:after, .modal-header:before, .nav:after, .nav:before, .navbar-collapse:after, .navbar-collapse:before, .navbar-header:after, .navbar-header:before, .navbar:after, .navbar:before, .pager:after, .pager:before, .panel-body:after, .panel-body:before, .row:after, .row:before {
    display: table;
    content: " ";
}
```

We could add some additional styling in ZAC to prevent/override the BrowZer styles, but I could see this happening on other webpages outside ZAC that have elements with class `.row` which is probably fairly common. I think the best solution would be to add some additional specificity to the rules in `ziti-browzer-css.css` to prevent conflicts with styles on other webpages.

**Comment by rentallect** [comment]
Fixed in `0.49.1`
